frontend/src/features/permits/pages/Application/components/review/ConfirmationCheckboxes.tsx

```diff
@@ -64,7 +66,8 @@ export const ConfirmationCheckboxes = ({
               }`
             }
             key={x.description}
-            checked={x.checked}
+            checked={isApplicationInReview ? true : x.checked}
+            disabled={isApplicationInReview}
```

Comment:
In the context of the ApplicationInQueueReview.tsx component, these checkboxes are disabled, since the staff user reviewing the application should not be making/editing the attestations themselves.